.changeset/sweet-swans-cry.md

```diff
@@ -0,0 +1,9 @@
+---
+"react-router": patch
+"react-router-dom": patch
+"react-router-dom-v5-compat": patch
+"react-router-native": patch
```

Comment:
Changesets is smart enough to bump dependent packages, so we only need to indicate the package in which changes were made

```suggestion
```

Reply:
Fixed, I guess I didn't read the `yarn changeset` prompts too closely 😆